### DISCUSSION TITLE
IR Toy v2 in 2024

### DISCUSSION THREAD

**Discussion by h3xstream** [bengtmartensson/IrScrutinizer#543] [Q&A] [answered]
Are there stores that still have the IR Toy in stock?
Any alternative device that does not require special assembly would be fine for my use case.

Thanks

**Comment by bengtmartensson** [comment, marked as answer]
[Irdroid USB](https://irdroid.com/irdroid-usb-ir-transceiver/) is a stripped-down clone of the IrToy, for a slightly higher price.

**Reply by h3xstream** [reply]
Thanks a lot
